tests/test_django_models.py

```diff
@@ -81,3 +81,9 @@ def test_serialization():
     fields = ser.getvalue()[0]["fields"]
     assert fields["color"] == m.color.value
     assert fields["taste"] == m.taste.value
+
+
+def test_nonunique_label():
```

Comment:
Just to be safe, could this test ensure the value gets written to (and loaded from) the database correctly?

```python
obj = MyModel.objects.create(labeled_enum=MyModel.LabeledEnum.FOOBAR)
assert obj.labeled_enum is MyModel.LabeledEnum.FOOBAR

obj = MyModel.objects.get(pk=obj.pk)
assert obj.labeled_enum is MyModel.LabeledEnum.FOOBAR
```

or something? :)